### PR TITLE
fix(createPositionManager): computePosition should not apply styles after destruction

### PR DESCRIPTION
**Pull request by ling1726**
Since `computePosition` is an async function, it can still resolve after the instance is destroyed.

Updates the destruction mechanism to have a [flag similar to poppver v2](https://github.com/floating-ui/floating-ui/blob/2893e9a8d2ebd895eb4311a8873afac62f75123e/src/createPopper.js#L171). This flag is used both on `forceUpdate` (to avoid memory leaks) and `computePosition` (to avoid applying outdated position).

### CODE REVIEW

**Review by layershifter — APPROVED** [review]

File: packages/react-components/react-positioning/src/createPositionManager.ts [hunk 1]
```diff
@@ -37,6 +37,7 @@ interface PositionManagerOptions {
  */
 export function createPositionManager(options: PositionManagerOptions): PositionManager {
   const { container, target, arrow, strategy, middleware, placement } = options;
+  let destroyed = false;
```
Comment (layershifter) [inline packages/react-components/react-positioning/src/createPositionManager.ts:40]:
```suggestion
  let isDestroyed = false;
```

nit: We use `is` prefix there already (isFirstUpdate)

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26593/perf-test-react-components/Avatar-mount_base.html">1321</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26593/perf-test-react-components/Avatar-mount.html">1335</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26593/perf-test-react-components/Button-mount_base.html">951</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26593/perf-test-react-components/Button-mount.html">947</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26593/perf-test-react-components/FluentProvider-mount_base.html">1628</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26593/perf-test-react-components/FluentProvider-mount.html">1626</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26593/perf-test-react-components/FluentProviderWithTheme-mount_base.html">629</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26593/perf-test-react-components/FluentProviderWithTheme-mount.html">643</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26593/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">599</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26593/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">600</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26593/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">631</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26593/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">632</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26593/perf-test-react-components/InfoButton-mount_base.html">566</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26593/perf-test-react-components/InfoButton-mount.html">583</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26593/perf-test-react-components/MakeStyles-mount_base.html">1900</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26593/perf-test-react-components/MakeStyles-mount.html">1874</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26593/perf-test-react-components/Persona-mount_base.html">2974</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26593/perf-test-react-components/Persona-mount.html">3020</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26593/perf-test-react-components/SpinButton-mount_base.html">2564</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26593/perf-test-react-components/SpinButton-mount.html">2565</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                                                                                             |      Baseline (minified/GZIP) |                            PR |                                                                                                                                                                                                Change |
| :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: | ----------------------------: | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox (including child components)</abbr>                                                 |  `82.678 kB`<br />`26.408 kB` |  `82.686 kB`<br />`26.411 kB` |  `8 B` <br />`3 B`  |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/ComboboxField.fixture.js'>ComboboxField</abbr>                                                                    |  `79.095 kB`<br />`26.183 kB` |  `79.103 kB`<br />`26.184 kB` |  `8 B` <br />`1 B`  |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Dropdown.fixture.js'>Dropdown (including child components)</abbr>                                                 |  `81.346 kB`<br />`26.144 kB` |  `81.354 kB`<br />`26.152 kB` |  `8 B` <br />`8 B`  |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> |  `188.355 kB`<br />`52.98 kB` | `188.363 kB`<br />`52.987 kB` |  `8 B` <br />`7 B`  |
| <samp>react-infobutton</samp> <br /> <abbr title='bundle-size/InfoButton.fixture.js'>InfoButton</abbr>                                                                        | `118.683 kB`<br />`36.127 kB` | `118.691 kB`<br />`36.133 kB` |  `8 B` <br />`6 B`  |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                                                          | `118.221 kB`<br />`36.568 kB` | `118.229 kB`<br />`36.575 kB` |  `8 B` <br />`7 B`  |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>                                             |  `121.29 kB`<br />`37.071 kB` | `121.298 kB`<br />`37.078 kB` |  `8 B` <br />`7 B`  |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr>                                                                                 | `105.773 kB`<br />`32.613 kB` |  `105.781 kB`<br />`32.62 kB` |  `8 B` <br />`7 B`  |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePositioning</abbr>                                                                    |   `20.856 kB`<br />`7.766 kB` |   `20.864 kB`<br />`7.776 kB` | `8 B` <br />`10 B`  |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                                                                                 |   `43.634 kB`<br />`15.26 kB` |  `43.642 kB`<br />`15.264 kB` |  `8 B` <br />`4 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                |          Size (minified/GZIP) |
| ---------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-alert</samp> <br /> <abbr title='bundle-size/Alert.fixture.js'>Alert</abbr>                                                                          |   `86.462 kB`<br />`21.76 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                                                                       |  `53.054 kB`<br />`14.661 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroup.fixture.js'>AvatarGroup</abbr>                                                             |    `15.42 kB`<br />`6.167 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroupItem.fixture.js'>AvatarGroupItem</abbr>                                                     |   `69.272 kB`<br />`18.96 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr> |  `60.588 kB`<br />`16.969 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>               |  `34.751 kB`<br />`11.453 kB` |
| <samp>react-persona</samp> <br /> <abbr title='bundle-size/Persona.fixture.js'>Persona</abbr>                                                                    |   `59.892 kB`<br />`16.55 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                            |    `6.069 kB`<br />`2.053 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/DataGrid.fixture.js'>DataGrid</abbr>                                                                    | `129.423 kB`<br />`35.243 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/PrimitivesOnly.fixture.js'>Table (Primitives only)</abbr>                                               |  `42.971 kB`<br />`11.781 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableAsDataGrid.fixture.js'>Table as DataGrid</abbr>                                                    | `127.672 kB`<br />`32.893 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSelectionOnly.fixture.js'>Table (Selection only)</abbr>                                            |  `81.106 kB`<br />`20.161 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSortOnly.fixture.js'>Table (Sort only)</abbr>                                                      |  `80.535 kB`<br />`20.005 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/0ac9dc52eb487b4bc0b5d059dadbceb7ac923773'>0ac9dc52eb487b4bc0b5d059dadbceb7ac923773</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

File: change/@fluentui-react-positioning-4bd82184-0e7b-46e8-a681-c87900b25568.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-positioning-4bd82184-0e7b-46e8-a681-c87900b25568.json:1]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 131 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26593&commitId=658cf2763a0883154956f5b53e25a8634751e3fa&env=prod)

✅ There was 0 screenshots added, 39 screenshots removed, 1701 screenshots unchanged, 0 screenshots with different dimensions and 92 screenshots with visible difference.


  <details><summary><b>unknown</b> 131 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Combobox Converged.Appearance- filled-darker.focused.chromium.png|242679| Changed|
|Combobox Converged.Appearance- filled-lighter.focused.chromium.png|242971| Changed|
|Combobox Converged.Appearance- outline (default).focused.chromium.png|243663| Changed|
|Combobox Converged.Appearance- underline.focused.chromium.png|243505| Changed|
|Combobox Converged.Disabled option.default.chromium.png|271979| Changed|
|Combobox Converged.Invalid- filled-darker.focused.chromium.png|242679| Changed|
|Combobox Converged.Invalid- filled-lighter.focused.chromium.png|242971| Changed|
|Combobox Converged.Invalid- outline.focused.chromium.png|243663| Changed|
|Combobox Converged.Invalid- underline.focused.chromium.png|243505| Changed|
|Combobox Converged.Open with grouped options.default.chromium.png|316262| Changed|
|Combobox Converged.Open with inlinePopup.default.chromium.png|271961| Changed|
|Combobox Converged.Open.default.chromium.png|271949| Changed|
|Combobox Converged.Option with long content.default.chromium.png|300082| Changed|
|Combobox Converged.Size- large.focused.chromium.png|244603| Changed|
|Combobox Converged.Size- small.focused.chromium.png|241871| Changed|
|Combobox Converged.With multiselect value.focused.chromium.png|243663| Changed|
|Combobox Converged.With placeholder.focused.chromium.png|243607| Changed|
|Combobox Converged.With value.focused.chromium.png|243531| Changed|
|Dropdown Converged.Disabled option.default.chromium.png|243818| Changed|
|Dropdown Converged.Open with grouped options.default.chromium.png|243818| Changed|
|Dropdown Converged.Open with inlinePopup.default.chromium.png|243831| Changed|
|Dropdown Converged.Open.default.chromium.png|243818| Changed|
|Dropdown Converged.Option with long content.default.chromium.png|243818| Changed|
|InfoButton.default - Dark Mode.active.chromium.png|9101| Changed|
|InfoButton.default - High Contrast.active.chromium.png|10983| Changed|
|InfoButton.default.active.chromium.png|31423| Changed|
|InfoButton.size.rest.chromium.png|87581| Changed|
|Menu Converged - basic.default - RTL.chromium.png|1607616| Changed|
|Menu Converged - basic.default.chromium.png|1607624| Changed|
|Menu Converged - groups.default - Dark Mode.chromium.png|1555773| Changed|
|Menu Converged - groups.default - High Contrast.chromium.png|1561185| Changed|
|Menu Converged - groups.default - RTL.chromium.png|1606761| Changed|
|Menu Converged - groups.default.chromium.png|1606770| Changed|
|Menu Converged - icon slotted content.default - RTL.chromium.png|1607408| Changed|
|Menu Converged - icon slotted content.default.chromium.png|1607422| Changed|
|Menu Converged - nested submenus.default - RTL.chromium.png|1607297| Changed|
|Menu Converged - nested submenus.default.chromium.png|1607317| Changed|
|Menu Converged - secondary content.default - RTL.chromium.png|1607476| Changed|
|Menu Converged - secondary content.default.chromium.png|1607493| Changed|
|Menu Converged - selection groups.default - RTL.chromium.png|1606715| Changed|
|Menu Converged - selection groups.default.chromium.png|1606760| Changed|
|Menu Converged - selection.checkbox - Dark Mode.chromium.png|1572902| Changed|
|Menu Converged - selection.checkbox - High Contrast.chromium.png|1579396| Changed|
|Menu Converged - selection.checkbox - RTL.chromium.png|1607616| Changed|
|Menu Converged - selection.checkbox.chromium.png|1607642| Changed|
|Menu Converged - split item.default - RTL.chromium.png|1607381| Changed|
|Menu Converged - split item.default.chromium.png|1607401| Changed|
|Menu Converged - submenuIndicator slotted content.default - RTL.chromium.png|1607874| Changed|
|Menu Converged - submenuIndicator slotted content.default.chromium.png|1607891| Changed|
|Menu nested within a MenuTrigger.default.chromium.png|1606820| Changed|
|Popover Converged.positioning - High Contrast.positioned popovers.chromium.png|65592| Changed|
|Popover Converged.positioning - RTL.positioned popovers.chromium.png|325426| Changed|
|Popover Converged.positioning.positioned popovers.chromium.png|325361| Changed|
|Positioning.arrow - RTL.chromium.png|1600858| Changed|
|Positioning.arrow.chromium.png|1600857| Changed|
|Positioning.auto size.chromium.png|1603723| Changed|
|Positioning.coverTarget - RTL.chromium.png|1603501| Changed|
|Positioning.coverTarget.chromium.png|1603515| Changed|
|Positioning.disable tether.chromium.png|1605494| Changed|
|Positioning.horizontal flip - RTL.chromium.png|1606144| Changed|
|Positioning.horizontal flip.chromium.png|1606144| Changed|
|Positioning.horizontal overflow - RTL.chromium.png|1606143| Changed|
|Positioning.horizontal overflow.chromium.png|1606143| Changed|
|Positioning.offset - RTL.chromium.png|1601543| Changed|
|Positioning.offset function - RTL.chromium.png|1601543| Changed|
|Positioning.offset function.chromium.png|1601543| Changed|
|Positioning.offset.chromium.png|1601543| Changed|
|Positioning.pinned.chromium.png|1606733| Changed|
|Positioning.position and align props - RTL.chromium.png|1601543| Changed|
|Positioning.position and align props.chromium.png|1601543| Changed|
|Positioning.position fixed - RTL.chromium.png|1601543| Changed|
|Positioning.position fixed.chromium.png|1601543| Changed|
|Positioning.reset target.chromium.png|1605082| Changed|
|Positioning.vertical flip.chromium.png|1606425| Changed|
|Positioning.vertical overflow.chromium.png|1606448| Changed|
|Positioning.visibility modifiers.has  [data-popper-escaped]  when the popper escapes the reference element's boundary.chromium.png|1594307| Changed|
|Positioning.visibility modifiers.has  [data-popper-is-intersecting]  when the popover intersects boundaries.chromium.png|1589132| Changed|
|Positioning.visibility modifiers.has  [data-popper-reference-hidden]  when the reference is hidden.chromium.png|1591186| Changed|
|Tooltip Converged.basic - Dark Mode.chromium.png|1582283| Changed|
|Tooltip Converged.basic - High Contrast.chromium.png|1583269| Changed|
|Tooltip Converged.basic.chromium.png|1608142| Changed|
|Tooltip Converged.inverted - Dark Mode.chromium.png|1582496| Changed|
|Tooltip Converged.inverted withArrow - Dark Mode.chromium.png|1582846| Changed|
|Tooltip Converged.inverted withArrow.chromium.png|1604981| Changed|
|Tooltip Converged.inverted.chromium.png|1606655| Changed|
|Tooltip Converged.positioning - High Contrast.positioned tooltips.chromium.png|40124| Changed|
|Tooltip Converged.positioning - RTL.positioned tooltips.chromium.png|205177| Changed|
|Tooltip Converged.positioning.positioned tooltips.chromium.png|205169| Changed|
|Tooltip Converged.text-wrapping.chromium.png|1607533| Changed|
|Tooltip Converged.withArrow - Dark Mode.chromium.png|1582336| Changed|
|Tooltip Converged.withArrow - High Contrast.chromium.png|1583260| Changed|
|Tooltip Converged.withArrow.chromium.png|1608014| Changed|
|Combobox Converged.Disabled option.active option.chromium.png|0| Removed|
|Combobox Converged.Disabled option.hover.chromium.png|0| Removed|
|Combobox Converged.Open with grouped options.active option.chromium.png|0| Removed|
|Combobox Converged.Open with grouped options.hover.chromium.png|0| Removed|
|Combobox Converged.Open with inlinePopup.active option.chromium.png|0| Removed|
|Combobox Converged.Open with inlinePopup.hover.chromium.png|0| Removed|
|Combobox Converged.Open.active option.chromium.png|0| Removed|
|Combobox Converged.Open.hover.chromium.png|0| Removed|
|Combobox Converged.Option with long content.active option.chromium.png|0| Removed|
|Combobox Converged.Option with long content.hover.chromium.png|0| Removed|
|Dropdown Converged.Disabled option.active option.chromium.png|0| Removed|
|Dropdown Converged.Disabled option.hover.chromium.png|0| Removed|
|Dropdown Converged.Open with grouped options.active option.chromium.png|0| Removed|
|Dropdown Converged.Open with grouped options.hover.chromium.png|0| Removed|
|Dropdown Converged.Open with inlinePopup.active option.chromium.png|0| Removed|
|Dropdown Converged.Open with inlinePopup.hover.chromium.png|0| Removed|
|Dropdown Converged.Open.active option.chromium.png|0| Removed|
|Dropdown Converged.Open.hover.chromium.png|0| Removed|
|Dropdown Converged.Option with long content.active option.chromium.png|0| Removed|
|Dropdown Converged.Option with long content.hover.chromium.png|0| Removed|
|Menu Converged - basic.default - RTL.hover menuitem.chromium.png|0| Removed|
|Menu Converged - basic.default.hover menuitem.chromium.png|0| Removed|
|Menu Converged - icon slotted content.default - RTL.hover menuitem.chromium.png|0| Removed|
|Menu Converged - icon slotted content.default.hover menuitem.chromium.png|0| Removed|
|Menu Converged - nested submenus.default - RTL.all open.chromium.png|0| Removed|
|Menu Converged - nested submenus.default.all open.chromium.png|0| Removed|
|Menu Converged - secondary content.default - RTL.hover menuitem.chromium.png|0| Removed|
|Menu Converged - secondary content.default.hover menuitem.chromium.png|0| Removed|
|Menu Converged - selection groups.default - RTL.selected.chromium.png|0| Removed|
|Menu Converged - selection groups.default.selected.chromium.png|0| Removed|
|Menu Converged - selection.checkbox - Dark Mode.selected.chromium.png|0| Removed|
|Menu Converged - selection.checkbox - High Contrast.selected.chromium.png|0| Removed|
|Menu Converged - selection.checkbox - RTL.selected.chromium.png|0| Removed|
|Menu Converged - selection.checkbox.selected.chromium.png|0| Removed|
|Menu Converged - split item.default - RTL.submenu open.chromium.png|0| Removed|
|Menu Converged - split item.default.submenu open.chromium.png|0| Removed|
|Menu Converged - submenuIndicator slotted content.default - RTL.submenus open.chromium.png|0| Removed|
|Menu Converged - submenuIndicator slotted content.default.submenus open.chromium.png|0| Removed|
|Menu nested within a MenuTrigger.default.submenu open.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv9"/>

File: packages/react-components/react-positioning/src/createPositionManager.ts [hunk 3]
```diff
@@ -97,9 +110,7 @@ export function createPositionManager(options: PositionManagerOptions): Position
   const updatePosition = debounce(() => forceUpdate());
 
   const dispose = () => {
-    // debounced update can still occur afterwards
-    // so destroy the reference to forceUpdate
-    forceUpdate = () => null;
+    destroyed = true;
```
Comment (ling1726) [inline packages/react-components/react-positioning/src/createPositionManager.ts:113]:
```suggestion
    isDestroyed = true;
```

File: packages/react-components/react-positioning/src/createPositionManager.ts [hunk 4]
```diff
@@ -37,6 +37,7 @@ interface PositionManagerOptions {
  */
 export function createPositionManager(options: PositionManagerOptions): PositionManager {
   const { container, target, arrow, strategy, middleware, placement } = options;
+  const isDestroyed = false;
```
Comment (ling1726) [inline packages/react-components/react-positioning/src/createPositionManager.ts:40]:
```suggestion
  let isDestroyed = false;
```

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Pull request merged**